examples/viewer.html

```diff
@@ -422,62 +422,53 @@
     };
 
     Viewer.prototype.processRequest_ = function(type, data, awaitResponse) {
-      log('Viewer.prototype.processRequest_');
-      if (type == 'documentLoaded') {
-        return this.documentReady_();
+      log('Viewer.prototype.processRequest_', type);
+      switch(type) {
+        case 'documentLoaded':
+          return this.documentReady_();
+        case 'requestFullOverlay':
+          return this.requestFullOverlay_();
+        case 'cancelFullOverlay':
+          return this.cancelFullOverlay_();
+        case 'pushHistory':
```

Comment:
FYI. replaceHistory is also a WIP @muxin. Though not really relevant to this cl.